Week-4/inheritance.js

```diff
@@ -0,0 +1,29 @@
+var Person = function() { };
+Person.prototype.initialize = function (name, age) {
+    this.name = name;
+    this.age = age;
+}
+
+// This as a object directly using this would work in straight forward way
+// let him = {
+//     __proto__: Person.prototype,
+//     teach: function (subject) {
+//         console.log(this.name + ' is now teaching ' + subject);
+//     }
+// }
+
+// now trying with Class Teacher 
+// this would create a class with constructor 
+class Teacher { }
+// assigning the function initialize to Teacher class
+Teacher.prototype.initialize = Person.prototype.initialize;
```

Comment:
@rajeshvlega9 Try to use setPrototypeOf method and dont directly set the prototype